src/main/scala/org/broadinstitute/hail/driver/VEP.scala

```diff
@@ -30,6 +31,9 @@ object VEP extends Command {
     @Args4jOption(name = "--force", usage = "Force VEP annotation from scratch")
     var force: Boolean = false
 
+    @Args4jOption(name = "--csq", usage = "Annotates with the VCF CSQ field instead of the JSON")
```

Comment:
how about "Annotates with the VCF CSQ field as a string, rather than the full nested struct schema"